analyses/ewings-cell-type/01-marker-gene-tumor-classification.Rmd

```diff
@@ -0,0 +1,516 @@
+First, let's just plot the cluster assignments alone.  
+
+These are clusters calculated in `scpca-nf` and its probable that these aren't the best clusters to use. 
+Separately, we will evaluate clustering for these samples, but for now we will look at the clusters we have. 
+
+```{r}
+# plot clusters on UMAP
+ggplot(marker_sum_exp, aes(x = UMAP1, y = UMAP2, color = cluster)) +
+  geom_point(size = 0.5, alpha = 0.5) +
+  theme_bw()
+```
+
+It looks like our large group of cells on the bottom right is comprised of a few clusters while most other groups of cells are a single cluster. 
+
+We can then look at marker gene expression across clusters.
+Below we visualize the total number of cells in each cluster that are classified as tumor or normal using the sum of marker gene expression. 
+Then for each set of cells, how many of those actually express each marker gene. 
+
+We expect to see two things: 
+
+- Tumor cells and normal cells tend to separate by cluster. 
+- Good marker genes will be expressed in tumor cells but not normal cells. 
+
+```{r, fig.height=10, fig.width=10}
+cluster_plot_df <- celltype_df |> 
+  dplyr::mutate(gene_expresed = dplyr::if_else(transformed_gene_expression > 0, TRUE, FALSE)) |> 
+  dplyr::select(gene_symbol, gene_expresed, sum_classification, cluster)
+
+ggplot(cluster_plot_df, aes(x = sum_classification, fill = gene_expresed)) +
+  geom_bar(stat = "count") +
+  facet_grid(rows = vars(gene_symbol),
+             cols = vars(cluster)) +
+  scale_fill_manual(values = c("grey", "blue")) +
+  theme_classic()
+  
+```
+
+Looking at this, most clusters are made up mostly of either normal or tumor. 
+Also, genes like ADGRG2 and HES1 do not seem specific to tumor cells. There are normal and tumor cells that express the gene and not very many tumor cells actually have expression of ADGRG2. 
+STEAP1 and NROB1 although not expressed in very many tumor cells, look to be specific to tumor cells which is promising. 
+
+CD99 and CCND1 are expressed in a high proportion of tumor cells but are also expressed in quite a few groups of normal cells. 
+This is consistent with the literature for CD99 that it is expressed in normal cells, just at a lower level than in Ewing's cells. 
+
+Almost all genes have at least some expression in normal cells. 
+Perhaps those cells should also be categorized as tumor cells, or they do have some baseline expression in normal cells which is possible. 
+
+Now we can classify the cells based on clustering and marker gene expression. 
+
+```{r}
+# plot marker gene expression across all clusters 
+ggplot(marker_sum_exp, aes(x = sum_exp, fill = cluster)) +
+  geom_density() +
+  facet_grid(rows = vars(cluster)) +
+  theme_bw()
+```
+
+Looking at this there are definitely clusters with higher sum expression of the marker genes than other cells. 
+If the cluster has a median expression > 0 then all cells in that cluster will get assigned as tumor cells. 
```

Comment:
This is where I start to get a bit uncomfortable with just using the initial clustering. If you are going to assign all cells in a cluster as tumor or not, then I would want the clusters to be something more meaningful than the defaults. I might just skip this for now, as I don't think it does anything more useful than the cell-based assignment, at least at this stage.